gobcore/typesystem/__init__.py

```diff
@@ -72,6 +72,23 @@
 ]
 
 
+def enhance_type_info(type_info):
+    """
+    Enhance type info with gob types
+
+    For every type attribute a corresponding gob_type attribute is set
+    that contains the GOB type class
+
+    :param type_info:
+    :return:
+    """
+    if type_info.get("type"):
+        type_info["gob_type"] = get_gob_type(type_info["type"])
+    if type_info.get("secure"):
```

Comment:
maybe check that it is a dict instead of a specific key. then it will work for all dicts recusively.

Reply:
Included the test on value type and added a test for it